File: android/app/src/main/java/com/bugbite/www/tf/activities/helper/ConfettiView.java
package com.bugbite.www.tf.activities.helper;


import com.github.jinatonic.confetti.ConfettiManager;

public interface ConfettiView {

    ConfettiManager generateOnce();
    ConfettiManager generateStream();
    ConfettiManager generateInfinite();

}

File: android/app/src/main/java/com/bugbite/www/tf/activities/helper/ConfettiView.kt
package com.bugbite.www.tf.activities.helper


import com.github.jinatonic.confetti.ConfettiManager

interface ConfettiView {

    fun generateOnce(): ConfettiManager
    fun generateStream(): ConfettiManager
    fun generateInfinite(): ConfettiManager

}
